vpr/src/device/rr_graph_view.h

```diff
@@ -248,6 +248,13 @@ class RRGraphView {
     inline const char* node_side_string(RRNodeId node) const {
         return node_storage_.node_side_string(node);
     }
+    /*@brief Get the switch used for the iedge'th edge from specified RRNodeId.*/
+    inline short edge_switch(RRNodeId id, t_edge_size iedge) const {
+        return node_storage_.edge_switch(id, iedge);
+    }
+    inline short edge_switch(const RREdgeId& edge) const {
```

Comment:
For the first API ``edge_switch(RRNodeId id, t_edge_size iedge)``, please comment

```
Get the switch id that represents the iedge'th outgoing edge from a specific node
TODO: We may need to revisit this API and think about higher level APIs, like ``switch_delay()``
```

Reply:
OK.  Surely I will do.